### PR TITLE
docs: update WSL instructions

### PR DESCRIPTION
**Pull request by ray-kraesig**
... with lessons learned from recent experimentation.

(Note that WSL is still not confidently supported, as we have no
automatic tests for this configuration.)

### CODE REVIEW

File: docs/howto/windows.md [hunk 1]
```diff
@@ -150,55 +140,66 @@ come and say hello [in #mobile on chat.zulip.org][czo]!
...
+    window using your right-click button. Ctrl + V will not work.)
 
-8. Install Node with the command below :
+8. Install Node v10 and Yarn.
 
-    ```
-    curl -sL https://deb.nodesource.com/setup_11.x | sudo -E bash -
-    sudo apt-get install -y nodejs
-    ```
+    `sudo snap install node --classic --channel=10`
```
Comment (andersk) [inline docs/howto/windows.md]:
```console
anders@VERTICAL-SYNC:~$ sudo snap install node --classic --channel=10
Interacting with snapd is not yet supported on Windows Subsystem for Linux.
This command has been left available for documentation purposes only.
```

Comment (ray-kraesig) [reply docs/howto/windows.md]: Nuts! :disappointed:

**Comment by ray-kraesig** [comment]
Fixed. (Also some minor edits.)

**Comment by gnprice** [comment]
Thanks for the updates! Merged.

**Pull request merged**